backend/backend.go

```diff
@@ -683,3 +685,62 @@ func (backend *Backend) HandleURI(uri string) {
 		backend.log.Warningf("Unknown URI scheme: %s", uri)
 	}
 }
+
+// PocketWidgetSignAddress returns an unused address and makes the user sign a message to prove ownership.
```

Comment:
document both return values more clearly, especially that the sig is returned in base64

Reply:
Done, hope it is fine :)